src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = 40 * days;
```

Comment:
I recommend writing down the numbers as separate variables so that they can be easily changed if necessary